Duplicating Data (#215)
gristlabs/grist-help: help/raw-data.md

![Raw Data List](images/raw-data/raw-data-list.png)

Click on a data table to open it. Note that in the creator panel the [widget type](page-widgets.md#page-widgets) cannot be changed. Renaming the widget also renames the data table. Because raw data is intended to show all data, columns cannot be hidden, either. However, columns can be rearranged, deleted, created, and modified. For creators, this view may make it easier to edit data structure, add [formulas](formulas.md), [conditional formatting](conditional-formatting.md), and so on.

![Raw Data View](images/raw-data/raw-data-lightbox.png)

## Duplicating Data

Tables can be duplicated from the Raw Data page. Click the three-dot icon to the right of the table you wish to duplicate then select 'Duplicate Table' from the menu.

![raw-data-duplicate-table](images/raw-data/raw-data-duplicate-table.png)

> By default, a duplicated table will only contain the table structure, not the data. If you wish to copy all data in addition to the table structure, be sure to check the box prior to clicking 'Save'. If duplicating data and access rules exist for the original table, those rules **will not** be duplicated. Only the document default rules wil apply to the copy.

Reviewer: ```suggestion
By default, a duplicated table will only contain the table structure, not the data. If you wish to copy all data in addition to the table structure, be sure to check the box prior to clicking 'Save'. If the original table has any access rules, those rules **will not** be duplicated. Only the document's default rules will apply to the copied table.
```

<span class="screenshot-large">*![raw-data-duplicate-data](images/raw-data/raw-data-duplicate-data.png)*</span>
{: .screenshot-half } 

> This will create a new, unlinked duplicate table. Meaning, if you update data in this new copy, the original table will not be updated. 

Reviewer: ```suggestion
The duplicate table is a new table that is not linked to the original. Meaning, if you update data in the copy, the original table will not be updated, and vice versa.
```

> Note that instead of duplicating tables, it's usually better to segment data using [linked views](linking-widgets.md). For example, if you have multiple clients that all have similar data sets, you could keep all data for all clients in a single table but with the added column 'Client' to segment the data.

Reviewer: I'm a bit confused by how the "Client" column segments the data. Also not clear how to segment with linked views. 
 Maybe we can brainstorm a clearer example.

> ***insert example as to when you would use this feature***

Reviewer: Oops I think you may have failed to commit some changes. :)
